Core/CoreTests/Configuration/ConfigTests.swift

```diff
@@ -73,6 +76,9 @@ class ConfigTests: XCTestCase {
         
         XCTAssertEqual(config.agreement.privacyPolicyURL, URL(string: "https://www.example.com/privacy"))
         XCTAssertEqual(config.agreement.tosURL, URL(string: "https://www.example.com/tos"))
+        XCTAssertEqual(config.agreement.cookiePolicyURL, URL(string: "https://www.example.com/cookie"))
+        XCTAssertEqual(config.agreement.dataSellContentURL, URL(string: "https://www.example.com/sell"))
+        XCTAssertEqual(config.agreement.supportedLanguages, ["es"])
```

Comment:
How about moving the AgreementConfig tests to a different file, like to AgreementConfigTests? Thoughts?